python/triton/language/extra/cuda/libdevice.py

```diff
@@ -1086,7 +1086,7 @@ def isinf(arg0, _builder=None):
         "", "", [arg0], {
             (core.dtype("fp32"), ): ("__nv_isinff", core.dtype("int32")),
             (core.dtype("fp64"), ): ("__nv_isinfd", core.dtype("int32")),
-        }, is_pure=True, _builder=_builder)
+        }, is_pure=True, _builder=_builder).to(core.int1, _builder=_builder)
```

Comment:
There is also `isfinited` below. Also should probably update amd libdevice to match.